charts/datadog/templates/agent-clusterchecks-rbac.yaml

```diff
@@ -13,6 +13,25 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "datadog.fullname" . }}-cluster-checks
     namespace: {{ .Release.Namespace }}
+{{- range $role := .Values.datadog.secretBackend.roles }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "datadog.fullname" $ }}-cluster-checks-read-secrets-{{ $role.namespace }}
+  namespace: {{ $role.namespace }}
+  labels:
+{{ include "datadog.labels" $ | indent 4 }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "datadog.fullname" $ }}-cluster-checks
+    apiGroup: ""
+    namespace: {{ $.Release.Namespace }}
+roleRef:
+  kind: Role
+  name: {{ template "datadog.fullname" $ }}-secret-reader-{{ $role.namespace }}
+  apiGroup: ""
+{{- end }} # end range $role := .Values.datadog.secretBackend.roles
```

Comment:
without this the cluster checks wont have access to k8s secrets if `.Values.datadog.secretBackend.roles` is used. The cluster check would only have access if the ClusterRole had access to all k8s secrets which some cluster operators may not want to allow.

Reply:
I think this is likely because it's copied from the [other file](https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/rbac.yaml#L191-L195), but could you remove the comment and add the apiGroup `rbac.authorization.k8s.io` ?

Reply:
ok applied that change too to the other file